python/grass/script/core.py

```diff
@@ -1709,44 +1718,38 @@ def create_location(
     desc=None,
     overwrite=False,
 ):
-    """Create new location
+    """Create new project
 
     Raise ScriptError on error.
 
-    :param str dbase: path to GRASS database
-    :param str location: location name to create
-    :param epsg: if given create new location based on EPSG code
-    :param proj4: if given create new location based on Proj4 definition
-    :param str filename: if given create new location based on georeferenced file
-    :param str wkt: if given create new location based on WKT definition
+    :param str path: path to GRASS database
```

Comment:
maybe add "if not given, project with name=name will be created in the current working directory" or something like that?

Reply:
path parameter is required, so the documentation should rather say that it's path to GRASS database and then name for project needs to be specified or directly to the project.

Reply:
i changed it to:

```
:param str path: path to GRASS database or project; if path to database, project
                     name must be specified with name parameter
```